extensions/wikia/Qualaroo/scripts/Qualaroo.js

```diff
@@ -70,5 +80,17 @@
 		createCookie('qualaroo_survey_submission');
 	});
 
+	if(ads.context.opts.sourcePointDetection) {
+		document.addEventListener('sp.blocking', function () {
```

Comment:
I wonder if this piece of code would fit better in the file that handles SP integration. We could require this file in there or something like that.